cli/cli/cmds_volume.go

```diff
@@ -339,7 +344,11 @@ func (c *CLI) initVolumeCmds() {
 			// volumes that are attached or can be attached should be valid
 			// candidates for a mount operation
 			result, err := c.lsVolumes(
-				args, volumeStatusAttached, volumeStatusAvailable)
+				args,
+				requestAttachments(
+					apitypes.VolumeAttachmentsMine,
+					apitypes.VolumeAttachmentsAttached,
```

Comment:
Should `VolumeAttachmentsAttached` be here? That would be volumes that are attached to other hosts, which I believe would be unavailable for mounting...

Is this need for pre-emption? (I'm assuming so)